source/_integrations/zwave_js.markdown

```diff
@@ -20,56 +20,66 @@ ha_codeowners:
 ha_domain: zwave_js
 ---
 
-This integration allows you to control a Z-Wave network via [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/).
+This integration allows you to control a Z-Wave network via the [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver and it's the successor of the [previous Z-Wave integration](https://www.home-assistant.io/integrations/zwave/) in Home Assistant, which is now considered deprecated.
 
 ## Requirements
 
-Controlling your Z-Wave network with the Z-Wave JS integration is split up into two parts:
+Controlling your Z-Wave network using the Z-Wave JS integration has the following requirements:
 
-1. The [Z-Wave JS Server](https://github.com/zwave-js/zwave-js-server) is the gateway between your Z-Wave USB stick and Home Assistant. You can run this server separately from Home Assistant so your Z-Wave mesh will keep running if you restart or stop Home Assistant. The Home Assistant Z-Wave JS integration connects to this server with a websocket connection.
+1. The [Z-Wave JS Server](https://github.com/zwave-js/zwave-js-server) is the gateway between your Z-Wave USB stick and Home Assistant, think of it like a virtual bridge/hub, running in software. You can run this server separately from Home Assistant so your Z-Wave mesh will keep running if you restart or stop Home Assistant. The Home Assistant Z-Wave JS integration connects to this server with a websocket connection. You need to run this Z-Wave server before you can use the integration.
 
-2. The Z-Wave JS integration in Home Assistant. This integration connects to the Z-Wave JS Server to retrieve the info from your Z-Wave network.
+2. [Supported Z-Wave dongle](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). The Z-Wave controller dongle should be connected to the same host as where the Z-Wave JS server is running. In the configuration for the Z-Wave server you need to provide the path to this stick, Z-Wave sticks will generally be /dev/ttyACM0 and GPIO hats will generally be /dev/ttyAMA0.
 
-### Supervisor managed installation
+3. A 16-byte **network key** used in order to connect securely to compatible devices. It is recommended that a network key is configured as security enabled devices may not function correctly if they are not added securely. You must provide this network key in the configuration part of the Z-Wave JS Server. For new installations, a default key will be auto generated for you.
```

Comment:
What is a 16-byte network key?

Reply:
Uhm, do we have some info about that that I cna use here. I couldn't find it in the current zwave docs either.

Reply:
Uhm, do we have some info about that that I can use here. I couldn't find it in the current zwave docs either.

Reply:
The reason I trigger on this, is because of the questions on the difference in the network key between Z-Wave JS & OZW.

Reply:
I would just explain it as 32 hexadecimal characters chosen at random to act as a secret to encrypt data between the controller and secure devices and then provide an example key: ec59967bd5e2d570619c3ff1fa15d16d (generated from random.org)

Reply:
We have a specific key for documentation purposes... it will be rejected by the add-ons.

`2232666D100F795E5BB17F0A1BB7A146` (that matches the old key as well).